src/scss/_base.scss

```diff
@@ -8,46 +8,64 @@
   box-sizing: border-box;
 }
 
-html,
-body {
-  height: 100%;
+body,
+html {
+  overflow-x: hidden;
```

Comment:
I had a different setup where `.full-bleed` was necessary but it looks like it is now redundant. I used `overflow-x: hidden` to prevent accidental overflow issues if they happen but I am now reading more about this and it seems that it is actually a bad practice. I will remove both!